src/usr/local/www/status_dhcp_leases.php

```diff
@@ -114,10 +114,10 @@ function remove_duplicate($array, $field) {
 /* this pattern sticks comments into a single array item */
 $cleanpattern = "'{ gsub(\"#.*\", \"\");} { gsub(\";\", \"\"); print;}'";
 /* We then split the leases file by } */
-$splitpattern = "'BEGIN { RS=\"}\";} {for (i=1; i<=NF; i++) printf \"%s \", \$i; printf \"}\\n\";}'";
+$splitpattern = "'BEGIN{ RS=\"#\";} {for (i=1; i<=NF; i++) printf \"%s \", \$i; printf \"}\\n\";}'";
```

Comment:
Reuse the same character for comments since it is already cleaned, so the awk parsing doesn't break.

Reply:
please take a look at https://redmine.pfsense.org/issues/3500#note-4
it may be less 'uid' field character dependent